src/test/java/com/solvd/eshop/AccountSectionTest.java

```diff
@@ -22,15 +22,18 @@ public void setupTest() {
     }
 
     @Test(testName = "verify that registration success with new email")
-    public void verifyValidRegistrationTest () {
+    public void verifyValidRegistrationTest() {
         HomePage homePage = new HomePage(driver);
         homePage.clickAccountMenu();
         homePage.clickLoginButton();
         homePage.clickRegistrationButton();
-        homePage.typeEmail("freppeumeproiffei-9158@yopmail.com");
+        homePage.typeEmail("freppeumeproiffei-9159@yopmail.com");
         homePage.clickContinueButton();
         By newSuccessRegistration = homePage.clickPrivacyButton();
         Assert.assertNotNull(newSuccessRegistration, "Registration isn't completed");
```

Comment:
In tests we use verification of success result by some text that appears on the screen or that some element/pop up appears on the screen. Please update test verification.

Reply:
resolved